amp-update-cache/server.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright 2015-2016, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const CacheRefresh = require('./cache-refresh');
+const express = require('express');
+const multer = require('multer'); // Use multer to parse muti-part forms.
+const upload = multer();
+const app = express();
+
+app.use(express.static('public'));
+
+app.post('/api/getCacheRefreshUrl', upload.array(), (req, res) => {
```

Comment:
nit: afaik it's best practice to use hyphens instead of camel case for URL segments s/getCacheRefreshUrl/get-cache-refresh-url/. Also: I find it a bit weird if a post request is named with a "get". Maybe: `/api/cache/calculate-update-url`